### ISSUE TITLE
build_artifact.csharp_ext_windows_x86 Timeout

### ISSUE THREAD

**Issue by gnossen** [grpc/grpc#26183]
```
2021-05-05 12:38:47,434 TIMEOUT: build_artifact.csharp_ext_windows_x86 [pid=6748, time=1800.1sec]
```

https://source.cloud.google.com/results/invocations/83842c4f-49af-4fae-bc72-806a8d8ada41/log

**Comment by drfloob** [comment]
https://github.com/grpc/grpc/pull/26179 <-- PR I filed this morning to increase the timeout. It has caused our nightly builds to fail as well.

**Comment by jtattermusch** [comment]
Fixed by #26179
